src/main/resources/openapi.yml

```diff
@@ -1807,77 +1807,70 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RestError'
+                $ref: '#/components/schemas/ProblemDetail'
         404:
           description: User not found.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RestError'
+                $ref: '#/components/schemas/ProblemDetail'
         500:
           description: Server error.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RestError'
+                $ref: '#/components/schemas/ProblemDetail'
 components:
   schemas:
-    RestError:
-      title: REST Error
+    ProblemDetail:
+      title: Problem Detail
       description: The schema for all error responses.
       type: object
       properties:
+        type:
+          title: Type
+          description: Full URL that originated the error response.
+          type: string
+          format: text
```

Comment:
**issue**: we have to use the`uri` format
I think you could use the schema of this repository: https://github.com/belgif/openapi-problem/blob/master/src/main/openapi/problem/v1/problem-v1.yaml
It looks like very complete

Reply:
Would it just be changing the value of the "format" property from "text" to "uri"?

Reply:
Yes, as a minimum. I think we can take inspiration from what @pvdbosch with his agreement

Reply:
I will implement this change in the "format" property and maybe a few more according to the @pvdbosch.

Reply:
Taking advantage of the contact with you @arey, I would like to receive some tips to improve myself in open-source contributions.
I would like to start contributing to larger projects in the Spring ecosystem such as the Spring-Boot Framework itself.

Reply:
Hi, OK to use from the belgif openapi-problem repository! License is the same (both Apache License 2.0), so no problem there.